run-container/README.md

```diff
@@ -0,0 +1,116 @@
+# How to run the container for the Single cell RNA Seq Snap workflow (ScRNASeqSnap)
+
+We have generated Dockerfile and Definition file that contain all tools, packages, and dependencies necessary to run the code and analyses modules in the `sc-rna-seq-snap` repository. These are customized for `Rstudio/R v4.4.0` and `Seurat v4.4.0`.
+
+
+## To use the container in an R interactive session on HPC:
+
+From an interactive node on HPC, the user can open their session. Please modify memory and resources as needed for the analysis module to run.
+```
+bsub -P hpcf_interactive -J hpcf_interactive -n 1 -q standard -R "rusage[mem=4G]" -Is "bash"
+```
+
+## Load specific version of Singularity
+
+Please note that a version of Singularity is installed by default on all the cluster nodes at St Jude HPC. 
+Otherwise the user needs to ensure and load Singularity module by running:
+```
+module load singularity/4.1.1
+```
+
+
+1. Clone the repository
+```
+git clone https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap.git
+```
+
+2. Pull the singularity container
+```
+singularity pull docker://wabuala/rstudio_4.4.0_seurat_4.4.0:latest
+```
+
+
+3. Start the singularity container
+
+a. To run from the terminal
+```
+bash run-terminal.sh
```

Comment:
This command works and starts the singularity container. However, when checking packages with run-test-packages.Rmd, the html report with the session info shows that BiocManager_1.30.25 is loaded instead of 1.30.23. R and Seurat version are correct (4.4.0 for both).

Reply:
Thank you for reporting this difference in the `BiocManager` package version. This versioning was related to the proper installation of certain packages. Since these are installed and loaded properly, I consider this minor, and with no real impact on the container.

Also, note that the minor-minor version is not an issue in general. As long as the minor version is matched, the software should be identical. So, as long as it's `1.30.X`, the behavior should be identical.